examples/monte_carlo/train_mc.py

```diff
@@ -0,0 +1,240 @@
+            )
+            plt.plot(
+                range(input_length, input_length + output_length),
+                v_labels[i, :].numpy(),
+                color="green",
+                label="ground truth",
+            )
+            plt.legend()
+            plt.xticks(range(0, input_length + output_length, 2))
+            plt.fill_between(
+                range(input_length, input_length + output_length),
+                (torch.add(v_outputs[i, :], voutput_std[i, :], alpha=1)).numpy(),
+                (torch.add(v_outputs[i, :], voutput_std[i, :], alpha=-1)).numpy(),
+                color="orange",
+                alpha=0.1,
+            )
+            # plt.plot(range(input_length, input_length + output_length), (torch.add(v_outputs[i, :], voutput_std[i, :], alpha=1)).numpy(), color='orange', alpha=.3, label="upper conf bound")
+            # plt.plot(range(input_length, input_length + output_length), (torch.add(v_outputs[i, :], voutput_std[i, :], alpha=-1)).numpy(), color='orange', alpha=.3, label="lower conf bound")
+            plt.show()
+            # time.sleep(5)
+            n += 1
+
+
+def evaluate_model(model, test_loader: DataLoader):
+    running_vloss_mse = 0.0
+    running_vloss_mae = 0.0
+    model.eval()
+    with torch.no_grad():
+        for _, vdata in enumerate(test_loader):
+            vinputs, vlabels = vdata
+
+            size_batch = vinputs.shape[0]
+            assert size_batch <= batch_size
+            # other = torch.range(0, vinputs.shape[1] - 1).unsqueeze(0).unsqueeze(-1).repeat(size_batch, 1, 1)
+            # other = torch.range(0, vinputs.shape[1] - 1).unsqueeze(0).unsqueeze(-1).repeat(size_batch, 1, 1) / vinputs.shape[1]
+            # assert vinputs.shape == (size_batch, input_length, 1)
+            # vlabels_masked = vlabels
+            # vlabels_masked[:, :, 0] = 0.
+            voutputs = model(vinputs, vlabels)
+            vloss_mse = torch.nn.MSELoss()(voutputs.squeeze(-1), vlabels[:, :, 0])
+            vloss_mae = torch.nn.L1Loss()(voutputs.squeeze(-1), vlabels[:, :, 0])
+            running_vloss_mse += vloss_mse
+            running_vloss_mae += vloss_mae
+
+    mse = running_vloss_mse / len(test_loader)
+    mae = running_vloss_mae / len(test_loader)
+    return mse, mae
+
+
+if __name__ == "__main__":
+    input_length = 50
+    output_length = 150
+
+    model = ClassicalMCDTransformer(
+        input_features=3,
+        output_features=1,
+        d_model=16,
+        nhead=1,
+        dim_feedforward=200,
+        num_encoder_layers=1,
+        num_decoder_layers=1,
+        final_activation=None,
+        layer_norm=True,
+        dropout=0.0,
+        gaussian=True,
+        weight_drop=True,
+        rate=0.5,
+        std_dev=0.5,
+        istrainablesigma=True,
+    )
+
+    print(model)
+    print(f"Number of parameters: {sum(p.numel() for p in model.parameters())}")
+
+    epochs = 50
+    batch_size = 20
+
+    data_train, data_test = get_data_electricity()
```

Comment:
The script should check, if the data is there and if it is not, download it or throw and error that explains how to get it.